apps/web/src/services/cost-explorer/budget/budget-detail/BudgetDetailPage.vue

```diff
@@ -128,21 +129,29 @@ const handleConfirmDelete = () => {
 <style lang="postcss" scoped>
 .page-title-wrapper {
 
-    .title-right-extra {
-        @apply inline-flex items-center;
-        line-height: 1.09375rem;
-        float: right;
-        .divider {
-            height: 0.875rem;
-            margin: 0 0.5rem;
+    .header-container {
+        /* custom design-system component - p-header */
+        &:deep(.heading-wrapper) {
+            @apply flex items-center;
```

Comment:
issue: design system style custom is the last thing we have to do.
![image](https://github.com/cloudforet-io/console/assets/26986739/309c8830-f159-46ce-b820-d97867e887c5)
![image](https://github.com/cloudforet-io/console/assets/26986739/a4894cc6-80a9-4b8e-a143-220677759953)

You can fix it without using deep by adding min-height to .title-right-extra.

Reply:
Holy.. Thank you genius.